src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/xgq_hwemu.h

```diff
@@ -35,12 +35,19 @@
  *
  */
 
-#pragma once
+#ifndef __XGQ_HWEMU_H__
+#define __XGQ_HWEMU_H__
 
-#include "xgq.h"
-
-#include <list>
 #include <boost/pool/object_pool.hpp>
+#include <condition_variable>
+#include <cstdint>
+#include <list>
+#include <mutex>
+#include <thread>
+
+#include "em_defines.h"
+#include "ert.h"
+#include "xgq.h"

```

Comment:
Last missing header is vector I think.

```
/home/soeren/git/stsoe/XRT.tmp/src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/xgq_hwemu.h:156:12: error: no template named 'vector' in namespace 'std' [clang-diagnostic-error]
      std::vector<uint32_t> sq_buf;
```

Reply:
Oh, sorry. I was using the compile command you suggested and it did not show any error. 

I just added the vector header.